docs/meta/versions.json

```
@@ -0,0 +1,4 @@
+{
+  "1.19.2": "https://cdn.jsdelivr.net/gh/Fabulously-Optimized/fabulously-optimized@main/Packwiz/1.19.2/pack.toml",
```

```suggestion
  "1.19.2": "https://raw.githubusercontent.com/Fabulously-Optimized/fabulously-optimized/main/Packwiz/1.19.2/pack.toml",
```

Is an external CDN really needed here?

---

This problem probably isn't affected here, but in general: https://stackoverflow.com/questions/62901066/raw-github-js-file-not-loading-direct-link-like-cdn

> Technically speaking, GitHub doesn't allow source code to be accessed from their site like a CDN...

However, this is a meta file, so it might be fine here. At least for now, let's make it consistent lol

---

I think that link explains how you can't _embed_ content from GitHub to a different site, which is true. But you of course can use the content in an external program like this :grin: